### PR TITLE
add blog for newly added service

### PR DESCRIPTION
**Pull request by erwinheitzman**
## Proposed changes

Added a blog for the introduction of a newly added service.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: website/blog/2019-12-05-wiremock.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  services: ['wiremock'],
+  // ...
+};
+```
+
+There are a lot more options that are available for configuration, for a full overview go [here](https://github.com/erwinheitzman/wdio-wiremock-service/blob/master/README.md#options).
+
+## Writing tests
+
+*wdio-wiremock-service* creates a directory (`./mock` by default) where creates two directories (`mappings` and `__files`). It will use the `mappings` directory to look for mock data.
```
Comment (klamping) [inline website/blog/2019-12-05-wiremock.md]: should that be `where it creates`? Also, i suggest changing 'two directories' to 'two subdirectories' to be clearer (assuming that's true)

File: website/blog/2019-12-05-wiremock.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+title: WireMock as a service
```
Comment (christian-bromann) [inline website/blog/2019-12-05-wiremock.md]:
```suggestion
title: WireMock as a Service
```

File: website/blog/2019-12-05-wiremock.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+author: Erwin Heitzman
+authorURL: http://github.com/erwinheitzman
+authorImageURL: https://avatars1.githubusercontent.com/u/15839059?s=80&v=4
+---
+
+# wdio-wiremock-service
```
Comment (christian-bromann) [inline website/blog/2019-12-05-wiremock.md]:
```suggestion
# The Wiremock Service
```

File: website/blog/2019-12-05-wiremock.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+We are pleased to announce that we now have a new mocking service for WebdriverIO called `wdio-wiremock-service`.
+
+## What can it do?
+
+This service helps you to run WireMock seamlessly when running tests with [WebdriverIO](https://webdriver.io). It uses the well known [Maven](https://mvnrepository.com/repos/central) repository to download the WireMock jar for you which is then automatically installed, started and stopped. Stay up to date by joining the community over at [Gitter](https://gitter.im/erwinheitzman/wdio-wiremock-service) to join others and for support.
```
Comment (christian-bromann) [inline website/blog/2019-12-05-wiremock.md]:
```suggestion
This service helps you to run WireMock seamlessly when running tests with [WebdriverIO](https://webdriver.io). It uses the well known [Maven](https://mvnrepository.com/repos/central) repository to download the WireMock jar for you which is then automatically installed, started and stopped. Stay up to date by joining the community over at [Gitter](https://gitter.im/erwinheitzman/wdio-wiremock-service) for help and support.
```

File: website/blog/2019-12-05-wiremock.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+A few things you can do with this service:
+
+- Automatically run WireMock alongside the WebdriverIO testrunner
```
Comment (christian-bromann) [inline website/blog/2019-12-05-wiremock.md]:
```suggestion
- Automatically run [WireMock](http://wiremock.org/) alongside the WebdriverIO testrunner
```

File: website/blog/2019-12-05-wiremock.md (hunk 1 above)
Comment (christian-bromann) [reply website/blog/2019-12-05-wiremock.md]: Maybe a sentence would be nice for what the `__files` directory is for if you mention it.

**Comment by erwinheitzman** [comment]
Added some examples and made some changes, could someone please check the latest changes (especially regarding the fixtures and mocks)?

**Pull request merged**